api/src/main/java/org/openmrs/Auditable.java

```diff
@@ -19,49 +17,9 @@
  * this interface.
  * 
  * @since 1.5
- * @see OpenmrsData
- * @see OpenmrsMetadata
+ * @see MutableOpenmrsData
+ * @see MutableOpenmrsMetadata
```

Comment:
Do `MutableOpenmrsData` and `MutableOpenmrsMetadata` classes exist outside of this PR? Or should these be references `Changeable`, `BaseChangeableOpenmrsData`, and `BaseChangeableOpenmrsMetadata` classes?

Reply:
Good catch! I had them and removed them and was should have updated these javadocs